### ISSUE TITLE
Receiving Address missing after transaction

### ISSUE THREAD

**Issue by jrsikors** [litecoin-project/litecoin#443]
### Describe the issue

Sorry if this issue has been beat to death -- couldn't find it with my Google skills

Sent some LTC in QT wallet a while back, partial transaction, from this address:
https://live.blockcypher.com/ltc/address/LS3aX8ksKbdBEaM9Z5XZhALXvY5aU5Azuj/

transaction worked fine, but the "change"? seemed to have gone to this address:
https://live.blockcypher.com/ltc/address/Lf7bmZQaDLcahXP9ADXrSgrnoi4XZhNYC8/

The 2nd address doesn't show up on my receiving addresses.  It does show up if i do a listaddressgroupings .. 

Good news is that I can dumpprivkey for it.

I'm about to transfer some more LTC and I'd like to keep it all in the same address... but I dont like the idea of sending to an address that isn't listed on my receiving addresses

### Expected behaviour
the new address with my funds should have showed up in my wallet as a receiving address?

### Actual behaviour
the new address isn't listed as a receiving address

### Machine specs:
- OS: Windows 10

![ss1](https://user-images.githubusercontent.com/31370541/36287694-fff19d44-127b-11e8-9e5a-37658d5aeeed.PNG)

![ss2](https://user-images.githubusercontent.com/31370541/36287700-05ba4910-127c-11e8-905a-6971bbf0fcdb.PNG)

![ss3](https://user-images.githubusercontent.com/31370541/36287705-09766aac-127c-11e8-8343-63f2a52068be.PNG)

**Comment by jrsikors** [comment]
I went ahead and created a new wallet .. imported private key .. and now that address 
https://live.blockcypher.com/ltc/address/Lf7bmZQaDLcahXP9ADXrSgrnoi4XZhNYC8/

is showing up as a receiving address in my new wallet.

I guess I'm curious if what happened was 'normal behavior' .. or if the address should have shown up as a receiving address after my transfer.

I went ahead and put labels on my receiving addresses ... in case having a null label was causing the problem?
